nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -312,6 +324,26 @@ in
         '';
       };
     };
+
+    services.zfs.zed.settings = mkOption {
+      type = with types; attrsOf (oneOf [ str int bool (listOf str) ]);
+      example = {
+        ZED_DEBUG_LOG = "/tmp/zed.debug.log";
+
+        ZED_EMAIL_ADDR = [ "root" ];
+        ZED_EMAIL_PROG = "mail";
+        ZED_EMAIL_OPTS = "-s '@SUBJECT@' @ADDRESS@";
+
+        ZED_NOTIFY_INTERVAL_SECS = 3600;
+        ZED_NOTIFY_VERBOSE = false;
+
+        ZED_USE_ENCLOSURE_LEDS = true;
+        ZED_SCRUB_AFTER_RESILVER = false;
+      };
+      description = ''
+        ZFS Event Daemon /etc/zfs/zed.d/zed.rc content
```

Comment:
Maybe this could link to the `zed` manpage

Reply:
How do you do this ?

Reply:
It's a mouthful, but `<citerefentry><refentrytitle>zed</refentrytitle><manvolnum>8</manvolnum></citerefentry>`